src/rpc/util.cpp

```diff
@@ -209,65 +210,52 @@ CTxDestination AddAndGetMultisigDestination(const int required, const std::vecto
     return dest;
 }
 
-class DescribeAddressVisitor : public boost::static_visitor<UniValue>
```

Comment:
Have you considered the "overload" approach from https://en.cppreference.com/w/cpp/utility/variant/visit (looks like it may be included in c++20, https://wg21.link/P0051)?

Reply:
That'd be dangerous because it'd allow for implicit conversions and non-exhaustive visitors. Bugs like #17924 would be harder to find if the overload approach was used.

Reply:
Understood, thanks for the explanation.